jobserver/templates/workspace_output_list.html

```diff
@@ -30,6 +32,47 @@
   </ol>
 </nav>
 
-{% include "outputs-spa.html" %}
+<div class="row">
+  <div class="col-lg-8 offset-lg-2">
+    <h3>Outputs</h3>
+
+    <p>
+      Listed below are the outputs for the {{ workspace.name }} workspace.
+    </p>
+
+    <ul class="list-group list-unstyled mb-4">
+      {% if user_can_view_all_files %}
+      <li class="list-group-item d-flex justify-content-between">
+        <div>
+          <a href="{{ workspace.get_current_outputs_url }}">Latest Files</a>
```

Comment:
I'd call both of these Outputs or Files, not mix an match. But see below for a more complete suggestion.